boards/native/nrf_bsim/doc/nrf54l15bsim.rst

```diff
@@ -89,5 +89,6 @@ ARM's TrustZone is not modeled in this board. This means that:
   can be labeled as restricted for secure or non secure access.
 * TF-M cannot be used.
 
-Note that the CRACEN peripheral is not modeled. The mbedTLS library can still be used
-but with a SW crypto backend.
+Note that the CRACEN peripheral is not modeled. But the mbedTLS library can be used
+with a SW crypto backend.
```

Comment:
better? In any case, "Mbed TLS" is the correct spelling (and only actual request for change)

```suggestion
Note that the CRACEN peripheral is not modeled, but the Mbed TLS library can still be used
with a SW crypto backend.
```

Reply:
Thanks @kartben I rephrased it a bit more.
I wanted to remove that "still" as we are not supposed to be using Mbedtls in the real nrf54l15dk target either.